src/components/reviews/reviewBreakdown.jsx

```diff
@@ -7,10 +7,11 @@ import StarRating from "../starRating.jsx";
 
 const ReviewBreakdown = ({ filters, metadata, toggleHandler, clearHandler }) => {
   var total = 0;
+  var percent = 0;
 
-  if (metadata.recommended) {
-  total = metadata.recommended[0] + metadata.recommended[1];
-  var percent = Math.round((metadata.recommended[1] / total) * 100);
+  if (metadata.recommended !== undefined && !(metadata.recommended[0] === 0 && metadata.recommended[1] === 0)) {
```

Comment:
I like this method of checking to make sure data is present. Maybe add a comment to let others know faster what is happening here.